lib/ui/auth_flow/create_profile_screen.dart

```diff
@@ -27,75 +33,133 @@ class _CreateProfileScreenState extends State<CreateProfileScreen> {
       resizeToAvoidBottomInset: true,
       body: SafeArea(
         child: SingleChildScrollView(
-          padding: const EdgeInsets.fromLTRB(24, 32, 24, 32),
+          padding: const EdgeInsets.fromLTRB(24, 32, 24, 0),
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.center,
             children: [
-              const Center(
-                child: Text(
-                  'Set up your profile',
-                  style: TextStyle(fontSize: 22, fontWeight: FontWeight.bold),
+              const Text(
+                'Setup Your Profile',
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontSize: 30,
+                  fontWeight: FontWeight.w500,
+                  color: AppColors.glitch800,
                 ),
               ),
               const SizedBox(height: 32),
```

Comment:
We could use a .h here too.... and other dimension instances (paddings, sizedBoxes, fonts)